src/org/isf/accounting/gui/PatientBillEdit.java

```diff
@@ -1156,7 +1161,11 @@ public void actionPerformed(ActionEvent e) {
 		}
 		return jButtonSave;
 	}
-	
+
+	private boolean hasNewPayments() {
+		return insert && payItems.size() > 0 || (payItems.size() - payItemsSaved) > 0;
```

Comment:
(insert && payItems.size() > 0) for better readability?

Reply:
Sure! thanks